recipe/meta.yaml

```
@@ -23,17 +24,9 @@ requirements:
     - python
     - pip
     - setuptools
+    - wheel
     - cython
-    - six
-    - py_stringsimjoin >=0.3.0
-    - cloudpickle
-    - pyparsing >=2.1.4
-    - scikit-learn >=0.18,<0.22.0a0
-    - xgboost  # [not win]
-    - pyqt
-    - requests
-    - ipython  # ==5.6
-    - numpy  # ==1.16.2
+    - numpy
```

```suggestion
    - numpy {{ numpy }}
```
That makes it a bit more explicit that the cbc version is used.